alvr/client_openxr/src/extra_extensions/body_tracking_fb.rs

```diff
@@ -52,12 +73,33 @@ impl BodyTrackerFB {
             next: ptr::null(),
             body_joint_set,
         };
+        let body_tracking_fidelity_props = super::get_props(
+            session,
+            system,
+            SystemPropertiesBodyTrackingFidelityMETA {
+                ty: *SYSTEM_PROPERTIES_BODY_TRACKING_FIDELITY_META,
+                next: ptr::null_mut(),
+                supports_body_tracking_fidelity: sys::FALSE,
+            },
+        )?;
         unsafe {
             super::xr_res((ext_fns.create_body_tracker)(
                 session.as_raw(),
                 &info,
                 &mut handle,
             ))?;
+
+            if body_tracking_fidelity_props.supports_body_tracking_fidelity == sys::TRUE
+                && prefer_high_fidelity
+            {
+                let request_body_tracking_fidelity: RequestBodyTrackingFidelityMETA =
+                    get_instance_proc(session, "xrRequestBodyTrackingFidelityMETA")?;
+
+                super::xr_res(request_body_tracking_fidelity(
+                    handle,
+                    BodyTrackingFidelityMode::High, // Low is default behavior
+                ))?;
+            }
```

Comment:
Should we instead explicitly set fidelity to low when fidelity extension is available but prefer_high_fidelity is false?

Reply:
We _can_ but it defaults to low anyway. Personally, I believe as-is is fine but if you want me to include the case I can.

Reply:
I think it's more correct to explicitly set to low if the extension is available.